swagger-spec/institutions/nodes_relationships.yaml

```diff
@@ -1 +1,76 @@
 # /institutions/{institution_id}/relationships/nodes/
+get:
+  summary: List all institution-node relationships
+  description: >-
+    Returns all nodes affiliated with an institution. Users that have an institutional
+    affiliation can add the affiliation to nodes that they have write access over. This
+    displays an institutional logo on the node page.
+
+  tags:
+    - Institutions
+  operationId: institutions_relationships_nodes
+  responses:
+    '200':
+      description: OK
+      schema:
+        type: array
+        items:
+        # It is not clear what this should be at the moment
+          $ref: /swagger-spec/nodes/definition.yaml
+      examples:
+        application/json:
+          data:
+          - type: nodes
+            id: api6f
+          - type: nodes
+            id: y9jdt
+          links:
+            self: https://api.osf.io/v2/institutions/cos/relationships/nodes/
+            html: https://api.osf.io/v2/institutions/cos/nodes/
+
+post:
+  summary: Add an institution-node relationship
```

Comment:
"Create" over "Add" for post endpoints. Also maybe "Create a relationship between an institution and a node"?